core/templates/services/literally-canvas-helper.service.spec.ts

```diff
@@ -0,0 +1,285 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for LiterallyCanvasHelperService.
+ */
+
+describe('LiterallyCanvasHelperService', function() {
+  var LiterallyCanvasHelperService = null;
+
+  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.inject(function($injector) {
+    LiterallyCanvasHelperService = $injector.get(
+      'LiterallyCanvasHelperService');
+  }));
+
+  it('should convert a rectangle shapeobject to svg tag', function() {
+    var rectShape = {
+      x: 142.5,
+      y: 96.5,
+      width: 12,
+      height: 29,
+      strokeWidth: 1,
+      strokeColor: 'hsla(0, 0%, 0%, 1)',
+      fillColor: 'hsla(0, 0%, 100%, 1)',
+      id: 'de569866-9c11-b553-f5b7-4194e2380d9f'
+    };
+    var actualSvgTag = '<rect id="rectangle-de569866-9c11-b553-f5b7-4194e23' +
+    '80d9f" x="143" y="97" width="12" height="29" stroke="hsla(0, 0%, 0%, 1' +
+    ')" fill="hsla(0, 0%, 100%, 1)" stroke-width="1"></rect>';
+    var svgTag = null;
+    svgTag = (
+      LiterallyCanvasHelperService.rectangleSVGRenderer(rectShape));
+    expect(svgTag).toBe(actualSvgTag);
+  });
+
+  it('should convert a ellpise shapeobject to svg tag', function() {
```

Comment:
"ellipse" is misspelled. Please proofread the changes in your PR more carefully before submitting for review.

Reply:
Done.